examples/common/python/worker/worker_details.py

```diff
@@ -77,11 +78,15 @@ def load_worker(self,input_str):
         self.data_encryption_algorithm = worker_data['dataEncryptionAlgorithm']
         self.verification_key = worker_data['workerTypeData']['verificationKey']
         self.encryption_key = worker_data['workerTypeData']['encryptionKey']
+        if worker_data['workerTypeData']['proofData']:
```

Comment:
Can you add a comment when proofData will be initialized and by which module

Reply:
Done